CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Contributing
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to the Jenkins Operator. These are mostly guidelines, not rules. 
+Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+In this project we appreciate any kind of contributions: code, documentation, design, etc.
+Any contribution counts, and the size does not matter!
+
+## Newcomers
+
+If you are a newcomer contributor and have any questions, please do not hesitate to ask in the `#jenkins-operator` [Slack](https://virtuslab-oss.slack.com) Channel.
+
+#### Table Of Contents
+
+[Code of Conduct](#code-of-conduct)
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [Atom Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. 
+
+## Commit Messages
+
+All commit messages should follow
```

Comment:
this seem a little bit excessive to me, commit message should be short, simple and to the point

we could point out some tips for beginners, like the closes/fixes feature of github and that the first line should be no more than 80 chars and IF someone has more to say they should feel free to do so

but I'm against the "wall of text" requirement as I strongly feel it will discourage contribution

Reply:
+1 to that. I will update this accordingly

Reply:
I have updated this, and it is up to the discretion of the commiter to do the necessities.
Can you check this again ?

Reply:
yes, I like this lighter, encouraging tone :)